migrations/20200927205755_user.js

```diff
@@ -0,0 +1,20 @@
+'use strict'
+
+const tableName = require('../models/user.js')
+
+exports.up = function (knex) {
+    return knex.schema.createTable(tableName, (table) => {
```

Comment:
```suggestion
const { tableName } = require('../models/user.js')

exports.up = function (knex) {
    return knex.schema.createTable(tableName, (table) => {
```

@herzizza88 the problem is actually here. `createTable` is expecting a `string` however, you seem to be passing in an object. Try to print out `tableName` to see! 

If you look into `.../models/user.js`, what you you are exporting from that file is: 

```
module.exports = {
    User,
    model: User,
    tableName
}
```
... so, you'll need to do extract the particular property from the object and pass that to knex. 

Also, read about [`module.exports`](https://nodejs.org/api/modules.html#modules_module_exports), default exports, named exports etc

Reply:
sorry this is a really stupid question.. how do you print the tableName? do i need to call that in the index.js?  i tried         document.write(tableName).. but it didnt work ... ( i know .. very amatuer move)

Reply:
ah.. `console.log(somethingToPrint)` .. `document` works to print inside the browser :)